ingest/src/test/java/com/microsoft/azure/kusto/ingest/E2ETest.java

```diff
@@ -84,9 +96,9 @@ class E2ETest {
     public static void setUp() {
         tableName = "JavaTest_" + new SimpleDateFormat("yyyy_MM_dd_hh_mm_ss_SSS").format(Calendar.getInstance().getTime()) + "_"
                 + ThreadLocalRandom.current().nextInt(Integer.MAX_VALUE);
-        principalFqn = String.format("aadapp=%s;%s", appId, tenantId);
+        principalFqn = String.format("aadapp=%s;%s", APP_ID, TENANT_ID);
 
-        ConnectionStringBuilder dmCsb = createConnection(System.getenv("DM_CONNECTION_STRING"));
```

Comment:
These names are unchanged, I just moved all references to environment variables to the top of the E2ETest class so that it is easier for newcomers to the codebase to see exactly which environment variables they need in order to run the E2ETest. Great care was taken to ensure none of the environment variable names changed.

Reply:
This was a QoL thing I did because as a new developer coming into this codebase **the E2ETest was completely un-runnable for me.** I had no idea what environment variables I needed nor the format that said variables needed to be in. This made the task of getting the E2ETest class tests to even **run** a monumental challenge, let alone get them to pass. When you spread usage of the environment throughout the entire test class it's very hard to see the requirements to run the tests, so that every test run I would be greeted by yet another new environment variable I hadn't seen before that would cause E2ETest to fail. Now they're all in one place and someone external like myself could easily see what they need, upfront, and not waste an hour trying to track down what environment variables are needed to run E2ETest.